models/istio_config.go

```diff
@@ -66,3 +67,10 @@ type ResourcesPermissions map[string]*ResourcePermissions
 
 // IstioConfigPermissions holds a map of ResourcesPermissions per namespace
 type IstioConfigPermissions map[string]*ResourcesPermissions
+
+var IstioConfigHelpMessages = map[string][]IstioConfigHelp{
+	"virtualservices": {
+		{ObjectField: "spec.hosts", Message: "The destination hosts to which traffic is being sent. Could be a DNS name with wildcard prefix or an IP address. Depending on the platform, short-names can also be used instead of a FQDN (i.e. has no dots in the name)"},
```

Comment:
Early comment.
I think these messages could come from the regular config.
Typically default hard-coded values may work just fine, but it could be nice if the user may overwrite them when needed.
(It won't change the code, just perhaps moving this to the config and organize it there, that's all)

Reply:
I updated the code with this in mind, sounds ok for the kiali administrators to customize the help they want to give to the users. Thanks @lucasponce .

Reply:
Oh, I see it was changed... I don't know why this would be in the user config. When would a user ever want to overwrite these? Would we even want them to?

Reply:
"_When would a user ever want to overwrite these? Would we even want them to?_"

Sorry to be late to the party, I just saw the notifications for this today when it closed., but having user help in a separate file is really a best practice for localization/translation.  That way the content can be translated without worrying about introducing a bug in the actual code.